todoapp/todos/views.py

```
@@ -1,25 +1,69 @@
+      }
+    ]
     """
 
+    permission_classes = [AllowAny]
+    serializer_class = todos_serializers.TodoDetailSerializer
+
+    def create(self, request, *args, **kwargs):
+        self.serializer_class = todos_serializers.TodoCreateSerializer
+        return super().create(request, *args, **kwargs)
+
+    def get_user(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_user`

(ANN201)